AjaxControlToolkit.Jasmine/Global.asax.cs

```diff
@@ -10,6 +11,7 @@ public class Global : System.Web.HttpApplication {
             RegisterRoutes(RouteTable.Routes);
             BundleConfig.RegisterBundles(BundleTable.Bundles);
 
+            AjaxFileUploadHelper.RootTempFolderPath = HttpContext.Current.Server.MapPath("~/Temp");
```

Comment:
`path.StartsWith("~")` ?
We'll have to do this anyway if we want to enable web.config way